core/src/main/java/org/testcontainers/utility/TestcontainersConfiguration.java

```diff
@@ -185,6 +185,10 @@ public boolean environmentSupportsReuse() {
         return Boolean.parseBoolean(getEnvVarOrUserProperty("testcontainers.reuse.enable", "false"));
     }
 
+    public boolean isComposeV2Enabled() {
+        return Boolean.parseBoolean(getEnvVarOrUserProperty("testcontainers.composev2.enable", "false"));
```

Comment:
why default is `false`? Isn't v2 enabled by default, if the binary is available? Or is it more like "force"? (then I suggest we rename it to `testcontainers.compose.force_v2`)

Reply:
The current implementation checks v2 binary if `withLocalCompose` is active, but will make use of the property for the containerized version (defaulting to v1 here).

I think we can be bold and go with v2 as default for the containerized version, which makes it easier to deprecate containerized v1 in the future.

Reply:
Agree, I didn't realize this was not consistent. So, now it has been updated to true.